roles/st2repos/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# tasks file for st2repos
+
+- name: Install prereqs
+  sudo: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - debian-archive-keyring
+    - apt-transport-https
+  
+- name: Add keys to keyring
+  sudo: yes
+  apt_key:
+    url: https://packagecloud.io/StackStorm/staging-stable/gpgkey
```

Comment:
You can change all urls from `staging-stable` to `stable`, since `1.4` is out and we say it's ready for production.

So repo url is: https://packagecloud.io/stackstorm/stable/

See: https://stackstorm.com/2016/04/21/goodbye-aio/